tests/theme.rs

```diff
@@ -31,13 +26,10 @@ fn get_error(msg: &'static str) -> Report {
 }
 
 #[test]
-fn test_error_backwards_compatibility(){
+fn test_error_backwards_compatibility() {
     setup();
     let error = get_error("test");
 
-
-
-
```

Comment:
Same as above. I should have used empty comments and document this better, I've now realized.